### DISCUSSION TITLE
tests/tests4blog/test2_math_test/

### DISCUSSION THREAD

**Discussion by giscus** [Linguage/Giscus#1] [Announcements]
# tests/tests4blog/test2_math_test/

https://peakhan618.github.io/tests/tests4blog/test2_math_test/

**Comment by Linguage** [comment]
wow, it's cool!

**Reply by lrylegendre** [reply]
awesome

**Reply by Linguage** [reply]
yes, really awesome!

**Reply by Linguage** [reply]
now, I am following you!

**Comment by Linguage** [comment]
## 数学公式

看起来对数学公式的支持还是蛮强大的。

## 评论功能
[Giscus](https://github.com/giscus/giscus/tree/main)挺好用

**Comment by Linguage** [comment]
$\pi$
